tasks/docs.yaml

```diff
@@ -5,6 +5,8 @@ tasks:
   - name: update
     description: updates the CLI docs
     actions:
+      - cmd: mkdir -p src/cmd/bin/
+      - cmd: touch src/cmd/bin/tofu
```

Comment:
@catsby I hate this but I'm not sure what the best way to handle this might be. Do you have any thoughts?

Context: 
The docs, schema, and unit tests error out if the file the `go:embedd` is reference doesn't exist. Here and elsewhere, I added these commands to ensure _something_ exists at that path.

I really don't like adding these series of commands in several places but at the same time I dont want to check-in a placeholder file at that path because it will make it SUPER easy to accidentally commit the binary itself and muckup our git history with a large file.

Reply:
Synced in person.